src/main/java/budgetbuddy/logic/Logic.java

```diff
@@ -24,6 +25,11 @@
      */
     CommandResult execute(String commandText) throws CommandException, ParseException;
 
+    /**
+     * Returns an unmodifiable view of the list of accounts.
+     */
+    ObservableList<Account> getAccountList();
+
```

Comment:
this probably shouldn't be here considering we already have AccountsManager that we can get the account list from already.